src/backend/expungeservice/test/test_file.py

```diff
@@ -0,0 +1,3 @@
+def test_run():
+  five = 5
+  assert five == 5
```

Comment:
I think the `tests` directory was at the same level as `src` as per our original directory structure plan.

Reply:
Actually I wasn't able import modules when tests were outside expungservice dir. Did you face the same issue?

Reply:
I did not try it outside the expungeservice dir. I believe the documentation stated that it needed to be at the root directory (which would be `expungeservice`) and named `test`. We could customize the configuration, but I believe the default configuration works for now.

Reply:
I think I miss-understood what you meant. Lets take a look at it next time we meet. I think the test directory should actually be under backend. So far I'm not running into issues with that configuration.